.github/workflows/test.yml

```diff
@@ -49,13 +44,8 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - run: |
-          sudo apt-get update \
-            && sudo apt-get remove msodbcsql17 -y \
-            && sudo apt-get remove buildah podman -y \
-            && sudo apt-get autoremove -y \
-            && sudo apt-get upgrade \
-            && sudo apt-get -y install podman dbus-x11 \
-            && podman version
+          sudo apt-get -y update
+          podman version
```

Comment:
Note: whether this was a one-off, but adding the `sudo apt-get -y install podman` here as well resulted in an e2e failure as we attempted to delete a container, which wasn't possible due to `device or resource busy`. Keeping this iteration for now, as we can rely on the podman version (3.0.0) we get by default in the ubuntu LTS release.